sdk/storage/azure-storage-blob-nio/src/main/java/com/azure/storage/blob/nio/AzureFileSystemProvider.java

```diff
@@ -1133,29 +1131,29 @@ void closeFileSystem(String fileSystemName) {
         this.openFileSystems.remove(fileSystemName);
     }
 
-    private String extractAccountName(URI uri) {
+    private String extractAccountEndpoint(URI uri) {
         if (!uri.getScheme().equals(this.getScheme())) {
             throw LoggingUtility.logError(this.logger, new IllegalArgumentException(
                 "URI scheme does not match this provider"));
         }
         if (CoreUtils.isNullOrEmpty(uri.getQuery())) {
             throw LoggingUtility.logError(this.logger, new IllegalArgumentException("URI does not contain a query "
-                + "component. FileSystems require a URI of the format \"azb://?account=<account_name>\"."));
+                + "component. FileSystems require a URI of the format \"azb://?endpoint=<account_endpoint>\"."));
         }
 
-        String accountName = Flux.fromArray(uri.getQuery().split("&"))
-                .filter(s -> s.startsWith(ACCOUNT_QUERY_KEY + "="))
+        String endpoint = Flux.fromArray(uri.getQuery().split("&"))
```

Comment:
nit: I don't think we need to use `Flux` here, instead we can use `Arrays.stream`